#include<iostream>

int main()
{
    try
    {

    }

    catch(int error_id)
    {
        
    }

    catch(...)
    {

    }

    
    return 0;
}